From 7edf3d7f1835883a6d553346e02023d43ceb40be Mon Sep 17 00:00:00 2001
From: Guochun Huang <hero.huang@rock-chips.com>
Date: Mon, 9 Nov 2020 12:58:21 +0800
Subject: [PATCH] arm64: dts: rockchip: rk3568-evb: set boot/always on for
 vcca1v8_image

which supplying for EDP/HDMI/MIPI/LVDS TX AVDD

Change-Id: If2442ae2e37a6b49d8371c8955c3f9bd863559ae
Signed-off-by: Guochun Huang <hero.huang@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3568-evb.dtsi | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3568-evb.dtsi b/arch/arm64/boot/dts/rockchip/rk3568-evb.dtsi
index ec3f73c4ae9a..2ec0f0b923c7 100644
--- a/arch/arm64/boot/dts/rockchip/rk3568-evb.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3568-evb.dtsi
@@ -891,6 +891,8 @@
 			};
 
 			vcca1v8_image: LDO_REG9 {
+				regulator-always-on;
+				regulator-boot-on;
 				regulator-min-microvolt = <1800000>;
 				regulator-max-microvolt = <1800000>;
 				regulator-name = "vcca1v8_image";
-- 
2.35.3

